components/illustration/component.json

```diff
@@ -1,5 +1,6 @@
 {
     "component": "illustration",
+	"isDeprecated": true,
```

Comment:
this should be `is-deprecated` based on the other key/value pairs.

Reply:
I went ahead and updated it.

Reply:
Thanks!